addons/web/static/src/js/framework/misc.js

```diff
@@ -172,6 +172,12 @@ function ReloadContext (parent, action) {
 }
 core.action_registry.add("reload_context", ReloadContext);
 
+function SuccessNotify (parent, action) {
```

Comment:
I think that the name implies a success, which is too restrictive.  What about NotifyUser?